api/base/urls.py

```diff
@@ -80,18 +80,4 @@
     url(r'^$', RedirectView.as_view(pattern_name=views.root), name='redirect-to-root', kwargs={'version': default_version}),
 ]
 
-# Add django-silk URLs if it's in INSTALLED_APPS
-if 'silk' in settings.INSTALLED_APPS:
-    urlpatterns += [
-        url(r'^silk/', include('silk.urls', namespace='silk')),
-    ]
```

Comment:
Anywhere we are using silk locally or on other non-production envs?

Reply:
Nope, silk is not being used anyway, and has no SQL tables.